contracts/core/AMM/SportsAMMV2RiskManager.sol

```diff
@@ -415,10 +432,18 @@ contract SportsAMMV2RiskManager is Initializable, ProxyOwned, ProxyPausable, Pro
                     "Risk per market and position exceeded"
                 );
                 require(!_isRiskPerGameExceeded(marketTradeData, marketRiskAmount), "Risk per game exceeded");
-                require(!_isInvalidCombinationOnTicket(_tradeData, marketTradeData, i), "Invalid combination detected");
+                require(
+                    _isSGP || !_isInvalidCombinationOnTicket(_tradeData, marketTradeData, i),
```

Comment:
Should we apply the same check for `checkRisks` method?

Reply:
I dont think so, because checkRisks is only used for fetching a quote, but in case of SGP the quote is fetched from the API/CL node, the contract method for getting the quote is not applicable.